### PR TITLE
Fixed sphinx documentation

### PR DESCRIPTION
**Pull request by andyndang**

### CODE REVIEW

File: src/whylogs/core/summaryconverters.py [hunk 1]
```diff
@@ -124,9 +124,9 @@ def histogram_from_sketch(
         counts = [n]
     else:
         # Include the max value in the right-most bin
-        end += abs(end) * (1e-7)
+        end += abs(end) * 1e-7
         # Include the right edge in the bin edges
-        n_buckets = min(math.ceil(n / HIST_AVG_NUMBER_PER_BUCKET), MAX_HIST_BUCKETS)
+        n_buckets = min(math.ceil(n / avg_per_bucket), max_buckets)
```
Comment (lalmei) [inline src/whylogs/core/summaryconverters.py]: why is this here ?

Comment (lalmei) [reply src/whylogs/core/summaryconverters.py]: maybe a rebase is needed.

Comment (andyndang) [reply src/whylogs/core/summaryconverters.py]: Yeah it looks to be the case

**Review by lalmei — APPROVED** [review]
LGTM
